### PR TITLE
Support building testing on fedora and quay

### PR DESCRIPTION
**Pull request by phracek**
Support building, testing postgresql-container on Fedora host.

Please check if build-and-push-action contains proper versions.
This pull request automatically pushes these versions: `11`, `12`, `13`.

https://github.com/sclorg/postgresql-container/pull/438

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test-openshift]

File: src/Dockerfile.fedora [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/{{ spec.latest_fedora }}/s2i-core:latest
+FROM quay.io/fedora/s2i-core:35
```
Comment (zmiklank) [inline src/Dockerfile.fedora:1]: What about using tag "latest" here? We would not have to rewrite the Dockerfiles everytime a new Fedora is released, if we use latest.

**Review by zmiklank — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-push.yml [hunk 2]
```diff
@@ -18,30 +18,52 @@ jobs:
...
             registry_namespace: "centos7"
             tag: "centos7"
+            suffix: "centos7"
             quayio_username: "QUAY_IMAGE_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_BUILDER_TOKEN"
           - dockerfile_path: "13"
             dockerfile: "Dockerfile.c9s"
             registry_namespace: "sclorg"
             tag: "c9s"
+            suffix: "c8s"
```
[the remark this answers is not in the document]
Comment (phracek) [reply .github/workflows/build-and-push.yml:42]: Grrr, typop.

File: src/Dockerfile.fedora (hunk 1 above)
Comment (phracek) [reply src/Dockerfile.fedora:1]: Definitely not. Although `s2i-core container` has the latest tag, which is not good, we would like to tell the user, which Fedora version is used. `Latest` tag means, nowadays `36`, after a Fedora release.

**Comment by fila43** [comment]
LGTM all added versions are present in fedora and changes make sense to me.

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
IT looks like problem with subscription

Let's try it one more time 
[test-all]

**Pull request merged**